views/pages/restaurantprofile.ejs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <%- include '../partials/header.ejs' %>
```

Comment:
`<%- include ('../partials/header.ejs') %>`
Small issue with the include statement ()